app/templates/_webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const webpack = require('webpack');
+
+module.exports = {
+	entry: './src/<%= componentName %>.js',
+	module: {
+		rules: [{
+			test: /\.js$/,
+			exclude: /(node_modules)/,
+			use: {
+				loader: 'babel-loader',
+				options: {
+					compact: false,
+					presets: ['babel-preset-es2015'],
+					plugins: ['babel-plugin-transform-node-env-inline']
+				}
+			}
+		}, {
+			test: /\.scss$/,
+			use: [{
+				loader: "style-loader"
+			}, {
+				loader: "css-loader"
+			}, {
+				loader: "sass-loader"
+			}]
+		}]
+	},
+	output: {<% if (buildFormat === 'amd') { %>
+		library: '<%= repoName %>/src/<%= componentName %>',
+		libraryTarget: '<%= buildFormat %>',
+		filename: './build/<%= buildFormat %>/<%= repoName %>/src/<%= componentName %>.js'<% } else { %>
+		library: 'metal',
+		libraryTarget: 'this',
+		filename: './build/<%= buildFormat %>/<%= kebabCaseName %>.js'<% } %>
+	},
+	plugins: [
+		new webpack.optimize.ModuleConcatenationPlugin()
```

Comment:
Yeah, I was just pointing that if someone tries to create entry points based on this generator the duplicated `IncrementalDOM` dependency will trip her up as seen in the referenced `metal.js` issue. This is might be a big _gotcha_, so we can either set it as default, or simply document it somewhere...